internal/security/config/command/proxy/adduser/command_test.go

```diff
@@ -79,38 +102,60 @@ func TestAddUserJWT(t *testing.T) {
 	require.NoError(t, err)
 
 	config.KongURL.Server = tsURL.Hostname()
-	config.KongURL.AdminPort, _ = strconv.Atoi(tsURL.Port())
+	config.KongURL.ApplicationPort, _ = strconv.Atoi(tsURL.Port())
 
-	// Act
+	// Setup command "addUser w/JWT"
 	command, err := NewCommand(lc, config, []string{
-		"--token-type", "jwt",
-		"--user", "someuser",
-		"--algorithm", "RS256",
-		"--public_key", "testdata/rsa.pub",
+		"--token-type", tokenType,
+		"--user", user,
+		"--algorithm", algorithm,
+		"--public_key", publicKey,
+		"--jwt", jwt,
```

Comment:
this won't work: the command treats `--jwt` as JWT file not the raw jwt string.
See the argument you added:
```go
flagSet.StringVar(&cmd.jwtFile, "jwt", "", "The file containing the JWT for use when accessing the Kong Admin API")
```

Reply:
@lenny-intel and I actually adjusted this in our pair programming session - so it's consistent throughout now. It is now the actual JWT instead of the file.